code/clue.js

```diff
@@ -1,59 +1,396 @@
+  age: 68,
+  image: 'assets/white.png',
+  occupation: 'Retired',
+  favorite: 'candle stick'
+};
+
+//CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
 
 const rope = {
-  name: 'rope',
-  weight: 10
-}
+  name: 'a rope',
+  weight: 10,
+  flexibility: 3,
+  color: '#7f4d33',
+  id: "rope",
+  image: 'assets/ropes1.jpg'
+};
+
+const knife = {
+  name: 'a knife',
+  weight: 1,
+  flexibility: 9,
+  color: '#2f6a49',
+  id: 'knife',
+  image: 'assets/knife.jpg'
+};
+
+const candleStick = {
+  name: 'a candle stick',
+  weight: 2,
+  flexibility: 7,
+  color: '#ddafd9',
+  id: 'candleStick',
+  image: 'assets/candlestick.jpg'
+};
+
+const dumbBell = {
+  name: 'a dumbbell',
+  weight: 20,
+  flexibility: 6,
+  color: '#9b2c15',
+  id: 'dumbBell',
+  image: 'assets/dumbbell.jpg'
+};
+
+const poison = {
+  name: 'poison',
+  weight: 0,
+  flexibility: 10,
+  color: '#156172',
+  id: 'poison',
+  image: 'assets/poison.jpg'
+};
+
+const axe = {
+  name: 'an axe',
+  weight: 12,
+  flexibility: 3,
+  color: '#7b497f',
+  id: 'axe',
+  image: 'assets/axe.jpg'
+};
+
+const bat = {
+  name: 'a bat',
+  weight: 3,
+  flexibility: 7,
+  color: '#abb782',
+  id: 'bat',
+  image: 'assets/bat.jpg'
+};
+
+const trophy = {
+  name: 'a trophy',
+  weight: 5,
+  flexibility: 3,
+  color: '#fce853',
+  id: 'trophy',
+  image: 'assets/trophy1.jpg'
+};
+
+const pistol = {
+  name: ' a pistol',
+  weight: 1,
+  flexibility: 9,
+  color: '#d1a93c',
+  id: 'pistol',
+  image: 'assets/pistol.jpg'
+};
+
+// OBJECTS FOR THE ROOMS (BECAUSE THEY HAVE PICTURES)
+const diningRoom = {
+  name: 'Dining Room',
+  image: 'assets/dining.jpg'
+};
+
+const conservatory = {
+  name: 'Conservatory',
+  image: 'assets/conservatory.jpg'
+};
+
+const kitchen = {
+  name: 'Kitchen',
+  image: 'assets/kitchen.jpg'
+};
+
+const study = {
+  name: 'Study',
+  image: 'assets/study.jpg'
+};
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
+const library = {
+  name: 'Library',
+  image: 'assets/library.jpg'
+};
 
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
+const billiardRoom = {
+  name: 'Billiard Room',
+  image: 'assets/billiard.jpg'
+};
 
+const lounge = {
+  name: 'Lounge',
+  image: 'assets/lounge.jpg'
+};
+
+const ballRoom = {
+  name: 'Ball Room',
+  image: 'assets/ballroom.jpg'
+};
+
+const hall = {
+  name: 'Hall',
+  image: 'assets/hall.jpg'
+};
+
+const spa = {
+  name: 'Spa',
+  image: 'assets/spa.jpg'
+};
+
+const livingRoom = {
+  name: 'Living Room',
+  image: 'assets/livingroom.jpg'
+};
+
+const observatory = {
+  name: 'Observatory',
+  image: 'assets/observatory.jpg'
+};
+
+const theater = {
+  name: 'Theater',
+  image: 'assets/theater.jpg'
+};
+
+const guestHouse = {
+  name: 'Guest House',
+  image: 'assets/guesthouse.jpg'
+};
+
+const patio = {
+  name: 'Patio',
+  image: 'assets/patio.jpg'
+};
+
+// STEP 2. NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
```

Comment:
Really clean code and good that you have used the comments to explain more detail.